### PR TITLE
Schedule mayor cache ticking to the next year rather than every 20 mins

### PR DESCRIPTION
**Pull request by Emirlol**
This should reduce the API calls for the mayor endpoint by quite a lot, as a year lasts 5 days and 4 hours and ticking once per year compared to every 20 mins is a big difference.

This PR also fixes the `SkyblockTime` update scheduling. It was multiplied by 24 instead of 20 as a typo, so it was updating late by 4 minutes each passing day.

One thing I'm not sure about is whether the API updates the current mayor instantly. This is most likely false, but then idk how much extra time should be waited before the API call.

### CODE REVIEW

**Comment by Fluboxer** [comment]
remind me, how do we currently handle jerry perks and won't it deepfry it?

**Comment by kevinthegreat1** [comment]
You can do `Edit` -> `Convert Indents` -> `To Spaces` in IntelliJ to convert `Utils` back into spaces.

**Comment by kevinthegreat1** [comment]
> > You can do `Edit` -> `Convert Indents` -> `To Spaces` in IntelliJ to convert `Utils` back into spaces.
> 
> Is there a need to, though?

I would appreciate it, this is causing a lot of merge conflicts and is difficult to review.

**Comment by kevinthegreat1** [comment]
Utils should be converted back into spaces. We really can't afford the merge conflicts rn. `Edit -> Convert Indents`.

File: src/main/java/de/hysky/skyblocker/utils/Utils.java [hunk 1]
```diff
@@ -479,27 +484,47 @@ private static void resetLocRawInfo() {
...
+                if (!json.get("success").getAsBoolean()) throw new RuntimeException("Request failed!"); //Can't find a more appropriate exception to throw here.
+                return json.get("mayor").getAsJsonObject().get("name").getAsString();
             } catch (Exception e) {
-                LOGGER.error("[Skyblocker] Failed to get mayor status!", e);
+                throw new RuntimeException(e); //Wrap the exception to be handled by the exceptionally block
+            }
+        }).exceptionally(throwable -> {
+            LOGGER.error("[Skyblocker] Failed to get mayor status!", throwable.getCause());
+            if (mayorTickRetryAttempts < 5) {
+                int minutes = 5 * (mayorTickRetryAttempts == 0 ? 1 : 2 << mayorTickRetryAttempts - 1); //5, 10, 20, 40, 80 minutes
```
Comment (kevinthegreat1) [inline src/main/java/de/hysky/skyblocker/utils/Utils.java:508]:
```suggestion
                int minutes = 5 * 1 << mayorTickRetryAttempts; //5, 10, 20, 40, 80 minutes
```

**Review by kevinthegreat1 — APPROVED** [review]

**Pull request merged**